.github/workflows/scripts/check-autobump.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+const path = require('path');
+
+module.exports = async ({github, context, core}, autobump_list) => {
+  // Parse event
+  const json = fs.readFileSync(process.env.GITHUB_EVENT_PATH, { encoding: "utf-8" })
+  const event = JSON.parse(json)
+  const pull = event.pull_request
+
+  // Fetch PR files
+  const files = await github.rest.pulls.listFiles({
+      ...context.repo,
+      pull_number: pull.number
+  })
+
+  const autobump_array = autobump_list.split(" ")
+
+  for (const file of files.data) {
+      if(autobump_array.includes(path.parse(file.filename).name)) {
+        github.rest.issues.createComment({
+          ...context.repo,
+          issue_number: pull.number,
+          body: `
+Hi! You have used \`brew bump-formula-pr\` on a formula that is on the autobump list.
```

Comment:
I think we should do somewhat the opposite here and instead prevent `brew bump-formula-pr` from opening the PR in the first place.

Reply:
if we do that, do you have suggestions how we keep testbot out of that restriction?

Reply:
I also thought about this. I did not think the technical aspect through on how to filter it out for testbot (but there might be solutions).

I was more thinking about other implications of the change:
`brew bump-formula-pr` has been part of the success of Homebrew, we have advertised it as an "entry-level" tool for first time contributors. So I am worried to make drastic changes here.

I'm unsure if there will be situations where we still need be able to open these kind of pull requests. Maybe to bump a formula for an urgent CVE, though a manual pull request would workaround the limitation.

Also; we can still revisit the strategy later on.

Reply:
We could do `gh api  -H "Accept: application/vnd.github+json" /user` and check what the login is. I think it's essential to avoid a "I know what I'm doing" flag, otherwise everyone will just use that.

Reply:
> if we do that, do you have suggestions how we keep testbot out of that restriction?

an undocumented environment variable or similar.

> I was more thinking about other implications of the change:
> `brew bump-formula-pr` has been part of the success of Homebrew, we have advertised it as an "entry-level" tool for first time contributors. So I am worried to make drastic changes here.

yeh, I think we definitely should make drastic changes there 😂 

> though a manual pull request would workaround the limitation.

Exactly 👍🏻 

> I think it's essential to avoid a "I know what I'm doing" flag, otherwise everyone will just use that.

Agreed.

Reply:
Have opened https://github.com/Homebrew/brew/pull/16664 and https://github.com/Homebrew/homebrew-core/pull/162841 for this.

Also on my list is to handle a maximum number of PRs that can be open for a given author.